agent/consul/peering_backend.go

```diff
@@ -70,6 +70,9 @@ func (b *PeeringBackend) GetServerAddresses() ([]string, error) {
 		}
 		addrs = append(addrs, node.Address+":"+grpcPortStr)
 	}
+	if len(addrs) == 0 {
+		return nil, fmt.Errorf("grpc port is not configured on any servers")
```

Comment:
How about something a bit more strongly worded? We don't want users to end up in a situation where some servers have a gRPC port specified but not all

```suggestion
		return nil, fmt.Errorf("a grpc bind port must be specified in the configuration for all servers")
```